scripts/paper/generate_stwo_phase12_shared_lookup_bundle_figure.py

```
@@ -0,0 +1,386 @@
+            rsvg = subprocess.run(
+                ["rsvg-convert", "-f", "png", "-o", str(tmp_png), str(svg_path)],
+                check=False,
+                capture_output=True,
+                text=True,
+            )
+        except FileNotFoundError:
+            tmp_png.unlink(missing_ok=True)
+            if fail_closed:
+                raise SystemExit(
+                    f"rsvg-convert is required to render {png_path} from {svg_path}"
+                ) from None
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)